### ISSUE TITLE
Navigation in pictures

### ISSUE THREAD

**Issue by mihaelmilea** [ubports/gallery-app#27]
I have many pictures and there is no way I can navigate to the pictures taken in a certain year, a certain month. Maybe the events feature could help with that but for me the events feature is unusable since it freezes - maybe because of so many pictures? In meego there was a slider on the right side of the photo gallery and you could go to a different month/year.

**Comment by TronFourtyTwo** [comment]
This is true, at the moment browsing trought medias is very bad. We are working on a new way media are organized ( with collections ) which should help there.
